packages/fetchai/skills/aries_faber/skill.yaml

```diff
@@ -0,0 +1,29 @@
+name: aries_faber
+author: fetchai
+version: 0.1.0
+description: The aries_faber skill implements the alice player in the aries cloud
+  agent demo
+license: Apache-2.0
+aea_version: '>=0.3.0, <0.4.0'
+fingerprint:
+  __init__.py: Qma8qSTU34ADKWskBwQKQLGNpe3xDKNgjNQ6Q4MxUnKa3Q
+  behaviours.py: QmZUDpEr933j3jSuYh4xqqznCmxwxHTDXH9W2MTYxMePwB
+  handlers.py: QmVfpn8TC74fnZxAWsE2R1tfSR99jTBBfKLWAaZeywDJPR
+fingerprint_ignore_patterns: []
+contracts: []
+protocols: []
+behaviours:
+  aries_demo_faber_behaviour:
```

Comment:
I recommend removing `_behaviour` and `_handler` below as this will just make the name a bit shorter. They are accessible in context under `self.context.behaviours.{name}`..so the `behaviour` part is unnecessary.

Reply:
Done